src/Microsoft.ML.CpuMath/AvxIntrinsics.cs

```diff
@@ -15,7 +15,9 @@
 using System.Runtime.Intrinsics;
 using System.Runtime.Intrinsics.X86;
 using Microsoft.ML.Internal.CpuMath.Core;
+#pragma warning disable CS8981 // The type name only contains lower-cased ascii characters. Such names may become reserved for the language.
```

Comment:
Do we still need this alias for anything?

Reply:
Its still used a bunch in that file.